mkdocs-project-dir/docs/howto.md

```diff
@@ -1,3 +1,8 @@
+---
+title: How To
+layout: docs
+---
+
```

Comment:
I saw other files having this header ... this one had it not ... If it is a leftover, we may not keep it ... but other similar headers should be removed as well ...

Reply:
In theory the `layout` section should be removed from all pages -- which is simple enough to do but should probably be done in one go on its own -- but I think some pages have intentional title overrides in those headers and those should stay.

Reply:
So, I will remove the lines above in the next commit / update for this PR ... as it messes up with the `How Do I?` thing that is now visible ... OK.